iFrame Widget/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+    <div id="content" style="width: 100%; height: 300px;">
+        <h1 id="1firstHeading">Mal sehen was passiert</h1>
+        <h1 id="firstHeading" class="firstHeading pagetitle page-header" style="overflow: visible;">Abendempfindung, K.523 (<a href="/wiki/Category:Mozart,_Wolfgang_Amadeus">Mozart, Wolfgang Amadeus</a>)</h1>
+    </div>
+    <iframe src="" width="100%" height="400px" name="iFrame" title="Widget" id="widget">
+
+    </iframe>
+<script>
+    function startUpWidget() {
+        //Transform URL to Information
+
+        //var url = window.location.href;
+
+        var url = "http://imslp.org/wiki/Category:Beethoven,_Ludwig_van";
+        var url = "http://imslp.org/wiki/Studien_im_Generalbass,_Contrapunkt_und_in_der_Compositionslehre_(Beethoven,_Ludwig_van)";
```

Comment:
Yes since I do not have the same URL like imslp.org for testing. But this is just for testing.